src/data/thunks.js

```diff
@@ -98,6 +99,27 @@ export function getProctoringSettings() {
   };
 }
 
+export function getExamAccessToken() {
+  return async (dispatch, getState) => {
+    const { exam } = getState().examState;
+    if (!exam.id) {
+      logError('Failed to get exam access token. No exam id.');
+      handleAPIError(
+        // todo: check what user error is here, change error?
+        { message: 'Failed to fetch exam access token. No exam id was found.' },
+        dispatch,
+      );
+      return;
+    }
+    try {
+      const examAccessToken = await fetchExamAccessToken(exam.id);
+      dispatch(setExamAccessToken({ examAccessToken }));
```

Comment:
I am saving the exam access token to the exam state so I can retrieve from the unit level.